docs/developer/typescript/01_gettingStarted/02_install/02_projects.md

```diff
@@ -0,0 +1,24 @@
+---
+sidebar_label: Sub Projects
+---
+# Installing and running sub projects
+Ethereal Engine can run sub-projects or 'locations'.   
```

Comment:
Locations isn't correct here. Projects will have scenes that can be exposed as 'locations.' Anyway I just think `Ethereal Engine can run sub-projects` is good here.

Reply:
no, this needs to be reworded to say "Ethereal Engine can be extended with projects`. These aren't "sub-projects", they are equivalent to "projects" in other engines, (except modular like npm packages - because they are those too)

Reply:
This is unedited text from Anselm's effort, afaik. I might be wrong, not really sure :thinking:
Will take a look when I reorganize the projects explanation for the guide.
Haven't reached this part yet.

Reply:
https://github.com/EtherealEngine/etherealengine-docs/commit/ad7b558f16e4855ba5a8fa8859cd8059f09acd00